frontend/talentsearch/src/js/components/myStatusForm/MyStatusForm.test.tsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import "@testing-library/jest-dom";
+import {
+  GetMystatusQuery,
+  WorkRegion,
+  JobLookingStatus,
+  Language,
+  ProvinceOrTerritory,
+  SalaryRange,
+} from "../../api/generated";
+import { render, screen, fireEvent, waitFor } from "../../tests/testUtils";
+import { MyStatusForm, MyStatusFormProps } from "./MyStatusForm";
+
+const renderMyStatusForm = ({
+  initialData,
+  handleMyStatus,
+}: MyStatusFormProps) => {
+  return render(
+    <MyStatusForm initialData={initialData} handleMyStatus={handleMyStatus} />,
+  );
+};
+
+const mockDataForIncompletForm: GetMystatusQuery | undefined = {
+  __typename: "Query",
+  me: {
+    __typename: "User",
+    id: "11",
+    jobLookingStatus: JobLookingStatus.ActivelyLooking,
+    firstName: "Shubi",
+    lastName: "Suresh",
+    email: "fff@gmaik.com",
+    telephone: "12345679000",
+    preferredLang: Language.En,
+    currentProvince: ProvinceOrTerritory.Alberta,
+    currentCity: "fgrtyuii",
+    lookingForEnglish: true,
+    lookingForFrench: true,
+    lookingForBilingual: true,
+    isGovEmployee: undefined,
+    locationPreferences: [WorkRegion.Atlantic],
+    wouldAcceptTemporary: false,
+    expectedSalary: [SalaryRange["50_59K"]],
+  },
+};
+const mockDataForCompletForm: GetMystatusQuery | undefined = {
```

Comment:
Did you know you can import storybook stories for testing?  This would allow you to leave all this mock data out of your test file.  Check out frontend/common/src/components/Pagination/Pagination.test.tsx for an example.

Reply:
Woah! I didn't know this either! 🤯

Reply:
Ok. I will check.

Reply:
Yes! It work fine. Pretty smart way!